MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -388,30 +389,30 @@ private static TranscodeReason GetTranscodeReasonForFailedCondition(ProfileCondi
         /// <returns>The normalized input container.</returns>
         public static string? NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
         {
-            if (string.IsNullOrEmpty(inputContainer))
+            if (!inputContainer.Contains(',', StringComparison.OrdinalIgnoreCase) || profile == null)
             {
-                return null;
+                return inputContainer;
```

Comment:
Yes but I'd argue previous behaviour was not correct since something like `mkv` would return `null` which for sure can't be the intended return value for à normalization method.

Reply:
Late to the party but the old behavior is necessary because for LiveTV channel the inputContainer is indeed null upon first access because the source was never probed before.